Package.version

```diff
@@ -0,0 +1 @@
+0.11.0
```

Comment:
Opting for a simple plain text file now. 
We can change to `json`, `plist`, or some other structured format when the need arises.